CleanSlate/events/base_married_life_events.txt

```diff
@@ -6,7 +6,7 @@
 #
 #############################################
 #
-# Written by Sara Wendel-�rtqvist
+# Written by Sara Wendel-Örtqvist
```

Comment:
Beware the encoding. CK2 files have to be encoded in `Windows 1252`. With this change, I suspect that all of the files have been changed to `UTF-8` and will need to be changed back